### PR TITLE
env files integrated

### PR DESCRIPTION
**Pull request by micahelinor**
process.env integrated

### CODE REVIEW

File: backend/app.js [hunk 1]
```diff
@@ -1,11 +1,22 @@
+const dotenv = require("dotenv");
+dotenv.config();
```
Comment (abaruah117) [inline backend/app.js]: do require("dotenv").config();

File: backend/app.js [hunk 2]
```diff
@@ -1,11 +1,22 @@
...
+
+/*const db2 = require("db");
```
Comment (abaruah117) [inline backend/app.js]: delete unused comments

File: backend/app.js [hunk 3]
```diff
@@ -1,11 +1,22 @@
...
+
 const express = require("express");
 const app = express();
 const bodyParser = require("body-parser");
 const port = 3001;
 const db = require("./AnnouncementsQueries");
 const cors = require("cors");
 const fetch = require("node-fetch");
-const { userId, basicAuth } = require("./config");
+//const { userId, basicAuth } = require("./config");
```
Comment (abaruah117) [inline backend/app.js]: delete useless comment

File: backend/orderquery.js [hunk 4]
```diff
@@ -1,13 +1,14 @@
+const dotenv = require("dotenv");
+dotenv.config();
```
Comment (abaruah117) [inline backend/orderquery.js]: require("dotenv").config();

**Review by abaruah117 — CHANGES_REQUESTED** [review]
looks good! small changes

**Review by abaruah117 — APPROVED** [review]
lgtm

**Review by trev4ev — APPROVED** [review]
looks good!

**Comment by trev4ev** [comment]
be sure to share the updated .env file in the slack channel!

**Comment by micahelinor** [comment]
> be sure to share the updated .env file in the slack channel!

already did

**Pull request merged**